packages/fields/src/types/Unsplash/README.md

```diff
@@ -0,0 +1,143 @@
+<!--[meta]
+section: field-types
+title: Unsplash Field Type
+[meta]-->
+
+# Unsplash Field Type
+
+> **Unsplash**
+> The internet’s source of freely useable images.
+> Powered by creators everywhere.
+
+- _[Unsplash.com](https://unsplash.com)_
+
+The Unsplash Field Type enables storing meta data from the Unsplash API and
+generating URLs to dynamically transformed images.
+
+## Usage
+
+```javascript
+const { Keystone } = require('@keystone-alpha/keystone');
+const { Unsplash } = require('@keystone-alpha/fields');
+
+const keystone = new Keystone(/* ... */);
+
+keystone.createList('Post', {
+  fields: {
+    heroImage: {
+      type: Unsplash,
+      accessKey: '...', // Get one from https://unsplash.com/developers
+      secretKey: '...',
+    },
+  },
+});
+```
+
+Will add the following to the GraphQL schema:
+
+```graphql
+type UnsplashUser {
+  unsplashId: String
+  username: String
+  name: String
+  # The user's URL on Unsplash
+  url: String
+  # The user supplied portfolio URL
+  portfolioUrl: String
+  bio: String
+  location: String
+}
+
+# Mirrors the formatting options [Unsplash provides](https://unsplash.com/documentation#dynamically-resizable-images).
+# All options are strings as they ultimately end up in a URL.
+input UnsplashImageFormat {
+  w: String
+  h: String
+  crop: String
+  fm: String
+  auto: String
+  q: String
+  fit: String
+  dpi: String
+}
+
+type UnsplashImage {
+  unsplashId: String
+  width: Integer
+  height: Integer
+  color: String
+  description: String
+  publicUrl: String
+  publicUrlTransformed(transformation: UnsplashImageFormat): String
+  user: UnsplashUser
+}
+
+input UserCreateInput {
+  # An Unsplash image ID
+  heroImage: String
+}
+
+type Post {
+  heroImage: UnsplashImage
+}
+
+type Mutation {
+  createPost(data: PostCreateInput): Post
+}
+```
+
+Create a new Post with an `Unsplash` type:
+
+```graphql
+mutation {
+  createPost(data: { heroImage: "bJHWJeiHfHc" }) {
+    heroImage {
+      __typename
+      unsplashId
+      width
+      height
+      color
+      description
+      publicUrl
+      publicUrlTransformed(transformation: { w: "100" })
+      user {
+        unsplashId: String
+        username: String
+        name: String
+        portfolioUrl: String
+        bio: String
+        location: String
```

Comment:
`: String` shouldn't be here right?

Reply:
Yeah, it comes back from the Unsplash API as a string.

Reply:
This is in an example of a mutation though, not the GraphQL types, it shouldn't be like that in the actual mutation right?

Reply:
🤦‍♂️ Sorry, my bad. You're right!

I was reading it thinking it was the types. Will get it fixed 👍

Reply:
Fixed 👍